sunxspex/tests/test_thermal.py

```diff
@@ -0,0 +1,82 @@
+    0.435291,    0.144041,    0.0484909,   0.0164952,   0.00567440,  0.00196711,  0.000685918,
+    0.000240666, 8.48974e-05, 3.00666e-05, 1.07070e-05, 3.81792e-06, 1.36722e-06, 4.91871e-07,
+    1.76929e-07, 6.39545e-08, 2.30593e-08, 8.38502e-09, 3.04271e-09, 1.10372e-09, 4.03399e-10,
+    1.47199e-10, 5.36175e-11, 1.96076e-11, 7.20292e-12, 2.64275e-12, 9.68337e-13, 3.54305e-13,
+    1.30357e-13, 4.81556e-14, 1.77739e-14, 6.55418e-15, 2.41451e-15, 8.88565e-16, 3.26635e-16,
+    1.20928e-16, 4.49286e-17, 1.66830e-17, 6.19118e-18, 2.29618e-18, 8.51055e-19, 3.15220e-19,
+    1.16669e-19, 4.31491e-20, 1.59455e-20, 5.90236e-21, 2.20614e-21, 8.24339e-22, 3.07924e-22,
+    1.14983e-22]) * u.ph / u.cm**2 / u.s / u.keV
+
+dist_scaled_spectrum_unit = u.ph / u.cm**2 / u.s / u.keV
+dist_unscaled_spectrum_unit = u.ph / u.s / u.keV / u.sr
+# Output spectrum from SSW given energy edges from 3 - 28 keV in 0.5keV bins with
+# no relative abundances and not scaled to the observer distance.
+expected_line_spectrum_E032805_6MK_EM1e44_RelAbunFe2_NotObserverScaled = np.array([
+    1.6399155e+25,   3.1898577e+24,   9.9494481e+23,   1.5177821e+23,   5.2238873e+21,   1.1521569e+17,
+    1.5242971e+22,   9.5837332e+21,   2.2554210e+20,   3.7478687e+15,   1.2419604e+15,   6.9386484e+13,
+    4.3706957e+09,   2973542.5,       234939.45,       0.0000000,       0.0000000,       0.0000000,
+    0.0000000,       0.0000000,       0.0000000,       0.0000000,       0.0000000,       0.0000000,
+    0.0000000,       0.0000000,       0.0000000,       0.0000000,       0.0000000,       0.0000000,
+    0.0000000,       0.0000000,       0.0000000,       0.0000000,       0.0000000,       0.0000000,
+    0.0000000,       0.0000000,       0.0000000,       0.0000000,       0.0000000,       0.0000000,
+    0.0000000,       0.0000000,       0.0000000,       0.0000000,       0.0000000,       0.0000000,
+    0.0000000,       0.0000000]) * dist_unscaled_spectrum_unit
+
+expected_line_spectrum_E032805_6MK_EM1e44_NoRelAbun_earth_20190522 = np.array([
+    0.073248975,     0.014247916,     0.0044440511,    0.00067793718,   2.3333176e-05,   2.5751346e-10,
+    3.4042361e-05,   2.1403499e-05,   5.0370664e-07,   8.3715751e-12,   2.7737142e-12,   1.5496721e-13,
+    1.9522280e-17,   1.3281716e-20,   1.0493879e-21,   0.0000000,       0.0000000,       0.0000000,
+    0.0000000,       0.0000000,       0.0000000,       0.0000000,       0.0000000,       0.0000000,
+    0.0000000,       0.0000000,       0.0000000,       0.0000000,       0.0000000,       0.0000000,
+    0.0000000,       0.0000000,       0.0000000,       0.0000000,       0.0000000,       0.0000000,
+    0.0000000,       0.0000000,       0.0000000,       0.0000000,       0.0000000,       0.0000000,
+    0.0000000,       0.0000000,       0.0000000,       0.0000000,       0.0000000,       0.0000000,
+    0.0000000,       0.0000000]) * dist_scaled_spectrum_unit
+
+
+def test_continuum_emission():
```

Comment:
I would also explicitly have a docstring included with these three tests that this is for benchmarking against IDL. I'd also include how the above arrays are calculated (maybe even give the line of IDL code used?). This can be replaced down the line, but I think to make it clear to others what exactly this is.

Reply:
Great idea.